ocaml/gencert/selfcert.ml

```diff
@@ -159,8 +165,18 @@ let xapi_pool ~uuid pemfile =
   in
   R.failwith_error_msg res
 
-let xapi_cluster ~cn =
-  let expiration = expire_never () in
+let xapi_cluster ?valid_from ~cn () =
+  let fist = "/tmp/fist_gencert_backdate" in
```

Comment:
Let's define this just once, on top of this file, so that gencert.ml can get it too?